mkdocs.yml

```diff
@@ -14,6 +14,7 @@ nav:
         - Requirements: getting_started/requirements.md
         - Install Ibexa DXP: getting_started/install_ibexa_dxp.md
         - Install on Ibexa Cloud: getting_started/install_on_ibexa_cloud.md
+        - Install on Ibexa Engage: getting_started/install_on_ibexa_engage.md
```

Comment:
```suggestion
        - Install Ibexa Engage: getting_started/install_on_ibexa_engage.md
```

Reply:
It doesn't fit in Getting started. It has nothing to do with running Ibexa DXP.

I don't see where it should be. Somewhere like Commerce or Customer management. It also sounds like related to the same meta-topic than Personalization and CDP.

Reply:
Seeing the User doc part, "Documentation > Ibexa Engage > Ibexa Engage" would work here too.